controllers/spacebindingrequest/spacebindingrequest_controller_test.go

```diff
@@ -79,6 +124,518 @@ func TestCreateSpaceBindingRequest(t *testing.T) {
 			require.EqualError(t, err, "unable to get the current SpaceBindingRequest: mock error")
 		})
 
+		t.Run("MasterUserRecord cannot be blank", func(t *testing.T) {
+			// given
+			badSBR := spacebindingrequesttest.NewSpaceBindingRequest("jane", "jane-tenant",
+				spacebindingrequesttest.WithMUR(""), // empty MUR
+				spacebindingrequesttest.WithSpaceRole("admin"))
+			member1 := NewMemberClusterWithClient(test.NewFakeClient(t, badSBR, sbrNamespace), "member-1", corev1.ConditionTrue)
+			hostClient := test.NewFakeClient(t, janeSpace, janeMur, base1nsTier)
+			ctrl := newReconciler(t, hostClient, member1)
+
+			// when
+			_, err = ctrl.Reconcile(context.TODO(), requestFor(badSBR))
+
+			// then
+			// an error should be returned
+			require.EqualError(t, err, "MasterUserRecord cannot be blank")
+			// spaceBindingRequest exists with config and finalizer
+			spacebindingrequesttest.AssertThatSpaceBindingRequest(t, badSBR.GetNamespace(), badSBR.GetName(), member1.Client).
+				HasSpecSpaceRole("admin").
+				HasSpecMasterUserRecord(""). // empty
+				HasFinalizer()
```

Comment:
verify that it has corresponding reason, status, and message set

Reply:
Done in https://github.com/codeready-toolchain/host-operator/pull/848/commits/0d24598c352faf223469ee1b913d9a6fc3993772

Thanks!